### DISCUSSION TITLE
'verdaccio' is not an internal or external command, nor is it a runnable program or batch file.

### DISCUSSION THREAD

**Discussion by ZzZzZzjw** [verdaccio/verdaccio#3365] [Q&A] [answered]
when I run 'verdaccio' in my terminal , It shows that 'verdaccio' is not an internal or external command, nor is it a runnable program or batch file。I searched the Internet for this error, but there seems to be no corresponding solution

**Comment by marckassay** [comment]
I'm getting the same. I have the following versions installed:

```
C:\Users\marck> node -v
v16.13.2
C:\Users\marck> npm -v
8.1.2
C:\Users\marck> nvm -v
1.1.9
```

It was working a few months ago, not sure what happened at this point.

**Comment by marckassay** [comment, marked as answer]
This command works `npm install -g verdaccio`. Perhaps the command that is currently documented, `npm install --location=global verdaccio`, conflicts with Powershell.
